UPGRADE-15.0.md

```diff
@@ -85,6 +85,7 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
                 protected readonly AdministratorActivityFacade $administratorActivityFacade,
         ```
     -   see #project-base-diff to update your project
+-   change limit for description in articles and blog articles to no limit ([#3046](https://github.com/shopsys/shopsys/pull/3046))
```

Comment:
there is no breaking change, you can remove this entry completely :slightly_smiling_face:

Reply:
👌 I removed entry completely. https://github.com/shopsys/shopsys/pull/3049/commits/a5b4f61537157eac92be21685b6367d9c69e17ea